api/src/main/java/io/lumeer/api/model/Collection.java

```diff
@@ -273,7 +273,6 @@ public void patch(final Collection resource, final Set<RoleType> roles) {
       }
       if (roles.contains(RoleType.TechConfig)) {
          setPurpose(resource.getPurpose());
-         setRules(resource.getRules());
```

Comment:
I believe this is what breaks tests...

Reply:
You are right but not in a way you think :)